src/modules/todo.js

```diff
@@ -0,0 +1,60 @@
+import trash from '../trash.svg';
+
+export default class Todo {
+  constructor() {
+    this.todoList1 = JSON.parse(localStorage.getItem('todoList')) || [];
+  }
+
+    newTodolist = () => {
+      const formToReset = document.getElementById('frm');
+      const newInput = document.getElementById('new-todo');
+      newInput.addEventListener('keypress', (e) => {
+        if (e.code === 'Enter') {
+          if (e.target.value.length > 0) {
+            const value = {
+              description: e.target.value,
+              isCompleted: false,
+              index: this.todoList1.length,
+            };
+            this.todoList1.push(value);
+            localStorage.setItem('todoList', JSON.stringify(this.todoList1));
+            formToReset.reset();
+          }
+        }
+        this.getList();
+      });
+    }
+
+    clearLocalStorage = () => {
+      const listToclear = this.todoList1.filter((todo) => !todo.isCompleted);
+      localStorage.setItem('todoList', JSON.stringify(listToclear));
+      this.todoList1 = listToclear;
+      window.location.reload();
```

Comment:
I think the content needs to be updated without reloading the page because it is a single-page app.
what do you think?

Reply:
Exactly I am going to try it as well